crates/sui/tests/cli_tests.rs

```diff
@@ -1872,29 +1863,8 @@ async fn test_package_management_on_upgrade_command() -> Result<(), anyhow::Erro
     .unwrap();
     let mut upgrade_pkg_path = tmp_dir.path().to_path_buf();
     upgrade_pkg_path.extend(["dummy_modules_upgrade", "Move.toml"]);
-    let mut move_toml = std::fs::File::options()
-        .read(true)
-        .write(true)
-        .open(&upgrade_pkg_path)
-        .unwrap();
     upgrade_pkg_path.pop();
 
-    let mut buf = String::new();
-    move_toml.read_to_string(&mut buf).unwrap();
-
-    // Add a `published-at = "0x<package_object_id>"` to the Move manifest.
-    let mut lines: Vec<String> = buf.split('\n').map(|x| x.to_string()).collect();
-    let idx = lines.iter().position(|s| s == "[package]").unwrap();
-    lines.insert(
-        idx + 1,
-        format!(
-            "published-at = \"{}\"",
-            package.reference.object_id.to_hex_uncompressed()
-        ),
-    );
-    let new = lines.join("\n");
-    move_toml.write_at(new.as_bytes(), 0).unwrap();
-
```

Comment:
This test `test_package_management_on_upgrade_command` previously performed an upgrade to check that we are writing the correct package management info to `Move.lock`. It previously used the same logic, relying on `Move.toml` to have the info.

Now, we remove the reliance on `Move.toml` to perform the upgrade in this test (and it ought to still succeed).